<?xml version="1.0"?>
<launch>
<param name="/use_sim_time" value="true" />
<!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find detect_dynamic)/view_pcl.rviz" /> -->
  <node pkg="detect_dynamic" type="detect_dynamic" name="detect_dynamic" output="screen">
    <param name="leaf_size" value="0.5" />
    <param name="occ_thresh" value="0.5" />
    <param name="t_window" value="50" />
    <param name="x_crop" value="1.0" />
    <param name="y_crop" value="1.0" />
    <param name="z_crop" value="1.0" />
  </node>
</launch>